.github/workflows/e2e.yaml

```diff
@@ -0,0 +1,32 @@
+name: Run e2e tests
+
+on:
+  push:
+    branches:
+      - "main"
+
+env:
+  NGROK_AUTHTOKEN: ${{ secrets.NGROK_AUTHTOKEN }}
+  NGROK_API_KEY: ${{ secrets.NGROK_API_KEY }}
+  RANCHER_HOSTNAME: loosely-settling-swan.ngrok-free.app
```

Comment:
do we want to leave this public?

Reply:
Thats a good point. We could just add it as a secret just in case

Reply:
Added both rancher password and hostname into secrets.